src/VisualStudio/Core/Def/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -226,6 +233,7 @@ public int FormatSpan(IVsTextLines pBuffer, VsTextSpan[] tsInSurfaceBuffer)
                             }
 
                             // Default, just use CRLF
+                            newLineEndingType = EOLTYPE.eolCRLF;
```

Comment:
> Shouldn't the default come from .editorconfig as opposed to being hardcoded to crlf?

No. The configuration from .editorconfig should always be intentionally ignored except for one singular case: all line endings in the entire file are being normalized to the configured value.

Reply:
If it turns out that this line gets hit specifically for the case where the file contains no line endings at all, this would perhaps work. However, this implementation was taken directly from the VS snippet engine behavior.